packages/clerk-js/src/ui/elements/FormButtons.tsx

```diff
@@ -26,7 +26,7 @@ export const FormButtons = (props: FormButtonsProps) => {
         // Should the default key come from userProfile?
         localizationKey={resetLabel || localizationKeys('userProfile.formButtonReset')}
         block={false}
-        onClick={navigateToFlowStart}
+        onClick={() => router.navigate('..')}
```

Comment:
Is this safe to be used in all scenarios? What about screens that are 2 levels deep, like the OTP codes section?

Reply:
You might be right, but since I found another bug as well, let's remove it for now and reexamine it later on.